ci/test_cpp.sh

```diff
@@ -38,5 +38,16 @@ rapids-logger "Run gtests"
 export GTEST_OUTPUT=xml:${RAPIDS_TESTS_DIR}/
 ./ci/run_ctests.sh -j20 && EXITCODE=$? || EXITCODE=$?;
 
+# Run basic example from librmm-example package
+if [ $EXITCODE -eq 0 ]; then
```

Comment:
What's the reason for making this "only run the examples if all the tests pass"? Do they take a long time and you're trying to save CI resources?

I think it'd be helpful to be able to see all of the logs all at once, so that the next commit pushed after a failing testing run has a better chance of succeeding.

If you find that convincing, can you please remove the `if [ $EXITCODE -eq 0 ];` guard?

Reply:
Good call. I also made this run all examples (all executables in `bin/examples/librmm`.